overlay/base/usr/lib/system/_os/cli/checkupdates.py

```
@@ -54,18 +54,17 @@ def command(args: Namespace) -> None:
     print("\n".join(updates))
-    if cast(bool, args.download):
-        if (
-            [x for x in updates if x.startswith(f"{baseImage()} ")]
-            and force
-            or pull_available()
-        ):
-            try:
-                pull()
+    if cast(bool, args.download) and (
+        [x for x in updates if x.startswith(f"{baseImage()} ")]
+        and force
```

Ruff (0.15.21)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)